src/core/messenger.ts

```diff
@@ -514,7 +510,6 @@ export const init = (
 	>,
 	errorHandler: MessengerErrorHandler,
```

Comment:
Do we still need to accept an `errorHandler` as a parameter here? From what I can see, [we always pass](https://github.com/search?q=org%3Aguardian+initmessenger&type=code) `reportError` from `utils/report-error`, so I'm not sure what this abstraction gives us. Is it just for testing purposes?